CRM/Upgrade/Incremental/php/FiveTwentyThree.php

```diff
@@ -109,4 +109,14 @@ public static function removeGooglePlusOption(CRM_Queue_TaskContext $ctx) {
     return TRUE;
   }
 
+  /**
+   * Preserve original core behaviour which is to hide activities that belong to cases
+   * from general activity views, e.g. the Activities tab on the contact records.
+   *
+   */
+  public static function setDefaultCivicaseShowCaseActivities(CRM_Queue_TaskContext $ctx) {
+    Civi::settings()->set('civicaseShowCaseActivities', 0);
+    return TRUE;
+  }
```

Comment:
I think you don't need this to set on upgrade as the code will take default from settings you added [here](https://github.com/civicrm/civicrm-core/pull/16360/files#diff-6419f840391ad2d06092d9dd08faff03R108).